cove_ocds/tests_functional.py

```diff
@@ -455,62 +308,3 @@ def mockflatten(input_name, output_name, *args, **kwargs):
         assert 'conversion Warnings' not in body_text
 
 
-@pytest.mark.parametrize(('prefix'), PREFIX_LIST)
-def test_URL_invalid_dataset_request(server_url, browser, prefix):
-    # Test a badly formed hexadecimal UUID string
-    browser.get(server_url + prefix + 'data/0')
-    assert "We don't seem to be able to find the data you requested." in browser.find_element_by_tag_name('body').text
-    # Test for well formed UUID that doesn't identify any dataset that exists
-    browser.get(server_url + prefix + 'data/38e267ce-d395-46ba-acbf-2540cdd0c810')
-    assert "We don't seem to be able to find the data you requested." in browser.find_element_by_tag_name('body').text
-    assert '360 Giving' not in browser.find_element_by_tag_name('body').text
-    #363 - Tests there is padding round the 'go to home' button
-    success_button = browser.find_element_by_class_name('success-button')
-    assert success_button.value_of_css_property('padding-bottom') == '20px'
-
```

Comment:
Are `test_URL_invalid_dataset_request` and `test_500_error` now only tested for 360, whereas before they were also tested for OCDS?